api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -735,4 +735,20 @@ public void storeProperties_shouldEscapeSlashes() throws Exception {
 		
 		assertThat(actual.toByteArray(), is(expected.toByteArray()));
 	}
+
+	@Test
+	public void copyFile_shouldCopyFile() throws IOException {
```

Comment:
please refer to https://wiki.openmrs.org/display/docs/Unit+Tests
for the testing style used within OpenMRS. there is a plugin which can help you create the scaffolds for each test.